bulk-utils/add_workflows.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+
+# Title: add bump.yml and build_and_test.yml if they don't already exist
+#
+# To be executed on workstation, via multi-gitter.
+# Assumes you've cloned this repo (meta) down and are running this script from the repo root like so: ./tools/add_workflows.sh
```

Comment:
I wonder if you can use:
```
repo_root=$(git rev-parse --show-toplevel)
```
that way we can run it anywhere?

Reply:
It seems that the git command will be in the context of the temporary clone that MG does.

```sh
INFO[0034] Script output: here-is-repo-root /tmp/multi-git-changer-3518522704  repo=ZOSOpenTools/zosncport
```

Outside of this, `meta/bin` is expected to be in $PATH, right?
Can `meta/bulk-utils` be added too?

Reply:
I see, then my suggestion won't work :)
Yeah, it could be added here: https://github.com/ZOSOpenTools/meta/blob/main/.env#L16 . But if it's installed via zopen install meta, then it should really be in the bin directory as we don't add other directories to PATH.

Reply:
If moving it to bin is ok, I'll rename the files in this PR to go there.

If someone gets meta from the releases page (as they start off), that will also contain the script, right?
Then the `source`-ing will add it to PATH, and the scripts will be discoverable?
Or is zopen install meta necessary?

Reply:
After the discussion, I think it makes sense to leave these tools outside of bin. The user can add it to their PATH when they need to use it for now. 
Perhaps at some point we can re-organize all of the dev tools and place them under one common dir like dev-utils or similar.